_includes/c11-cpp11-and-beyond-and-toolchains.md

```diff
@@ -204,12 +221,30 @@ matrix:
       
 You can find the `clang` version shipped by Xcode [here](https://trac.macports.org/wiki/XcodeVersionInfo).
 
+### Clang on FreeBSD
+
+FreeBSD ships with Clang 8.0.1
```

Comment:
Would this be an appropriate place to mention that the default compiler on FreeBSD is Clang? People not familiar with FreeBSD might assume it's GCC like on Linux.

Reply:
sure, will add this here :-)